Rabbit/Rabbit/Rabbit.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Rabbit
+{
+    public enum MammalType
+    {
+        Rabbit,
+        Moves,
+        Sleeps,
+        Eats,
+    }
+
+    class Rabbit
+    {
+        public string Name { get; }
+        public string Eyes { get; }
```

Comment:
astea trebuie sa fie enum=-uri, care sa aiba valorile pe care le-am specificat in enunt